devel/smake/Portfile

```
@@ -1,41 +1,96 @@
 }
 
-livecheck.url       https://sourceforge.net/projects/s-make/files/
-livecheck.regex     ${name}-(\\d+(?:\\.\\d+)*)${extract.suffix}
+patchfiles              001-Makefile-Patch.diff
+
+post-patch {
+                        reinplace -locale C "s|/opt/schily|${prefix}|g" \
+                        ${worksrcpath}/DEFAULTS/Defaults.darwin \
+                        ${worksrcpath}/DEFAULTS/Defaults.mac-os10 \
+                        ${worksrcpath}/DEFAULTS_ENG/Defaults.darwin \
+                        ${worksrcpath}/DEFAULTS_ENG/Defaults.mac-os10 \
+                        ${worksrcpath}/libfind/find.c \
+                        ${worksrcpath}/libfind/find_main.c \
+                        ${worksrcpath}/librscg/scsi-remote.c \
+                        ${worksrcpath}/patch/tests/random/cmptest.sh \
+                        ${worksrcpath}/patch/tests/random/gentest.sh \
+                        ${worksrcpath}/smake/job.c \
+                        ${worksrcpath}/smake/make.c \
+                        ${worksrcpath}/smake/smake.1 \
+                        ${worksrcpath}/TEMPLATES/Defaults.gcc \
+                        ${worksrcpath}/TEMPLATES/Defaults.clang \
+                        ${worksrcpath}/TEMPLATES/Defaults.xcc
+}
+
+conflicts_build-append  cdrtools
+
+# https://trac.macports.org/ticket/31616
+compiler.blacklist      llvm-gcc-4.2 macports-llvm-gcc-4.2 {clang < 300}
+
+configure.ldflags-append -lintl
+
+universal_variant       no
+
+depends_build-append    port:gmake \
+                        port:gettext-runtime
+                        
+depends_lib-append      port:gettext-runtime
+
+build.type              gnu
+                        
+build.args-append       INS_BASE="${destroot}${prefix}" \
+                        INS_RBASE="${destroot}${prefix}" \
+                        DEFOSINCDIRS="${prefix}/include" \
+                        CC_OPT=${configure.optflags}
+
+destroot.destdir        INS_BASE="${destroot}${prefix}" \
+                        INS_RBASE="${destroot}${prefix}" \
+                        MANDIR=man \
+                        DEFINSUSR=${install.user} \
+                        DEFINSGRP=${install.group}
+
+use_parallel_build      no
+
+destroot {
+                        set smakePath [ glob -directory ${worksrcpath}/smake/OBJ */smake ]
+                        xinstall ${smakePath} ${destroot}${prefix}/bin
+
+                        set smakeManPath [ glob -directory ${worksrcpath}/smake/OBJ/ */*/smake.1 ]
+                        xinstall ${smakeManPath} ${destroot}${prefix}/share/man/man1
+
+                        set makefilesManPath [ glob -directory ${worksrcpath}/man/man5/OBJ/ */*/makefiles.5 ]
+                        xinstall ${makefilesManPath} ${destroot}${prefix}/share/man/man5
+
+                        set makerulesManPath [ glob -directory ${worksrcpath}/man/man5/OBJ/ */*/makerules.5 ]
+                        xinstall ${makerulesManPath} ${destroot}${prefix}/share/man/man5
```

```suggestion
    xinstall -m 0755    {*}[glob ${worksrcpath}/smake/OBJ */smake] \
                        ${destroot}${prefix}/bin

    xinstall -m 0644    {*}[glob ${worksrcpath}/smake/OBJ */*/smake.1] \
                        ${destroot}${prefix}/share/man/man1

    xinstall -m 0644    {*}[glob ${worksrcpath}/smake/OBJ /man/man5/OBJ/ */*/makefiles.5] \
                        ${destroot}${prefix}/share/man/man5
```
a bit cleaner and keeps permissions ;)

---

No problem.  If you like I can add permissions.

---

But I also need to add:

```
xinstall -m 0644    {*}[glob ${worksrcpath}/man/man5/OBJ/*/*/makerules.5] \
                        ${destroot}${prefix}/share/man/man5
```

---

Honestly you may simplified it to `{*}[glob ${worksrcpath}/man/man5/OBJ/*/*/*.5]` :)